sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -49,14 +57,18 @@ struct mc_private {
 
 struct codec_info {
 	int id;
-	bool direction[2]; // playback & capture support
+	int count;
```

Comment:
count of what? can we use a less ambiguous name?

Reply:
done